samples/nrf9160/modem_shell/README.rst

```diff
@@ -44,10 +44,31 @@ Enable and configure eDRX::
 
    link edrx -e -m --edrx_value 0010 --ptw 0010
 
-Enable and configure PSM::
+Subscribe for modem TAU and sleep notifications, enable and configure PSM::
 
+   link tau --subscribe
+   link msleep --subscribe
    link psm -e --rptau 01100100 --rat 00100010
 
+Disable autoconnect::
+
+   link nmodeauto --disable
+   
+   Reboot the DK and it remains in pwroff functional mode and link is not connected.
```

Comment:
Is this text coming from the code? If not, you might want to have it in normal text.

Reply:
Modifying to example description